aws_lambda_powertools/utilities/validation/base.py

```diff
@@ -28,8 +39,10 @@ def validate_data_against_schema(data: Union[Dict, str], schema: Dict, formats:
         When JSON schema provided is invalid
     """
     try:
-        formats = formats or {}
-        fastjsonschema.validate(definition=schema, data=data, formats=formats)
+        formats = get_field_or_empty_dict(formats)
+        handlers = get_field_or_empty_dict(handlers)
+        provider_options = get_field_or_empty_dict(provider_options)
```

Comment:
```suggestion
        formats = formats or {}
        handlers = handlers or {}
        provider_options = provider_options or {}
```

I see why you did that, but because it's built-in and it's a short statement, we can keep relying on Python's expressiveness here over a function, as there's not much else besides an `or`.

Reply:
Reverted.